device/bgpsegmentweight.yaml

```diff
@@ -0,0 +1,18 @@
+components:
+  schemas:
+    Device.BgpSegmentWeight:
+      x-status: under-review
+      description: >-
+        Optional container for BGP SR TE Policy segment weight settings.
+      type: object
+      properties:
+        enable:
```

Comment:
Since we now support assymetric configuration natively in the revised model , the enable field might not be needed. The presence of this control was needed mostly to create assymetric configurations in a symmetric model . Now the subtlv can be selectively added only where required.